sources/packages/backend/libs/services/src/disbursement-overaward/disbursement-overaward.service.ts

```diff
@@ -70,4 +75,58 @@ export class DisbursementOverawardService {
     }
     return result;
   }
+
+  /**
+   * Get all overawards which belong to a student.
+   * @param studentId student.
+   * @returns overaward details of a student.
+   */
+  async getOverawardsByStudent(
+    studentId: number,
+  ): Promise<DisbursementOveraward[]> {
+    return this.disbursementOverawardRepo.find({
+      select: {
+        createdAt: true,
+        originType: true,
+        overawardValue: true,
+        disbursementValueCode: true,
+        creator: { firstName: true, lastName: true },
+        studentAssessment: {
+          id: true,
+          application: { applicationNumber: true },
+          triggerType: true,
+        },
+      },
+      where: {
+        student: { id: studentId },
+      },
+      relations: {
+        creator: true,
+        studentAssessment: { application: true },
+      },
+    });
+  }
+
+  /**
+   * Add a manual overaward deduction.
+   * @param awardValueCode award value code.
+   * @param overawardValueDeducted overaward deducted value.
+   * @param auditUserId user who added overaward deduction
+   * @returns overaward record created.
+   */
+  async addManualOverawardDeduction(
+    awardValueCode: string,
+    overawardValueDeducted: number,
+    studentId: number,
+    auditUserId: number,
+  ): Promise<DisbursementOveraward> {
+    const overawardManualRecord = new DisbursementOveraward();
+    overawardManualRecord.creator = { id: auditUserId } as User;
+    overawardManualRecord.disbursementValueCode = awardValueCode;
+    overawardManualRecord.overawardValue = -overawardValueDeducted;
```

Comment:
I would expect these values to come from the UI ready. The API should just accept it (this would require a change on the class validator as it is right now also).
Either way, while doing it on the UI I believe that we should get the number and multiply it by `-1` to invert it.